### ISSUE TITLE
Error occurs in motion planner optimizer

### ISSUE THREAD

**Issue by EcustBoy** [MCZhi/DIPP#19]
Hi~ author,  I try to change the cost terms based on your default  design, but when I start run it, during gradient backward step in theseus,  the following type of error will occur:
**There was an error while running the linear optimizer. Original error message: torch.linalg.cholesky: For batch 0: U(X,X) is zero, singular U.. Backward pass will not work. To obtain the best solution seen before the error, run with torch.no_grad()**
(X means number unfixed number, e.g. 2)

And I found  this error always occurs as long as  I remove **acc_cost** or **steering_cost** ,  but it runs normally when I remove any other cost terms,  

I would like to ask if you have encountered this type of error during research?  Is the cost function specifically designed to avoid such error?   I'll thanks much for your reply~

**Comment by MCZhi** [comment]
Hi, @EcustBoy. I haven't run into the problem before but you may try another optimizer such as cholmod_sparse_solver.

**Comment by EcustBoy** [comment]
> Hi, @EcustBoy. I haven't run into the problem before but you may try trying another optimizer such as cholmod_sparse_solver.

Thanks for your advice~  p.s.  I also wanna know which version of theseus you use? not sure whether it's related to the library version :-)

**Comment by MCZhi** [comment]
I am using 0.1.2, should be working fine.
